### PR TITLE
fix: misc params

### PR DESCRIPTION
**Pull request by 0xArdi**
This PR fixes various issues with the params.

### CODE REVIEW

File: packages/balancer/services/autonomous_fund/service.yaml [hunk 1]
```diff
@@ -22,6 +22,8 @@ models:
       allowed_lp_addresses: ${ALLOWED_LP_ADDRESSES:list:["0x87cc0d34f6111c8A7A4Bdf758a9a715A3675f941"]}
       enforce_allowlist: ${ENFORCE_ALLOWLIST:bool:true}
       observation_interval: ${OBSERVATION_INTERVAL:int:600}
+      service_registry_address: '0x48b6af7B12C71f09e2fC8aF4855De4Ff54e775cA'
+      on_chain_service_id: ${ON_CHAIN_SERVICE_ID:int:2}
       consensus:
```
Comment (DavidMinarsch) [inline packages/balancer/services/autonomous_fund/service.yaml:27]: why not `share_tm_config_on_startup` too? Let's bring this to parity with el coll imo

File: packages/balancer/skills/autonomous_fund_abci/models.py [hunk 2]
```diff
@@ -108,7 +108,7 @@ def setup(self) -> None:
 class Params(
     LiquidityProvisionParams,
     PoolManagerParams,
-    TransactionParams,
```
Comment (DavidMinarsch) [inline packages/balancer/skills/autonomous_fund_abci/models.py:111]: Why did you remove these? Because already defined in Termination?

Comment (0xArdi) [reply packages/balancer/skills/autonomous_fund_abci/models.py:111]: Yes, its a bit awkward having to do this. Maybe it makes more sense to remove the tx settlement params from the termination params.

Comment (0xArdi) [reply packages/balancer/skills/autonomous_fund_abci/models.py:111]: Although then not setting tx settlement params means you will get the error when the termination tx is submitted.

File: packages/balancer/services/autonomous_fund/service.yaml (hunk 1 above)
Comment (0xArdi) [reply packages/balancer/services/autonomous_fund/service.yaml:27]: Yes, makes sense.

File: packages/balancer/skills/liquidity_provision_abci/models.py [hunk 3]
```diff
@@ -71,3 +71,14 @@ def _ensure_managed_pool_address(  # pylint: disable=no-self-use
...
+    def _ensure_multisend_address(  # pylint: disable=no-self-use
+        self, kwargs: Dict
+    ) -> str:
+        """Ensure that the MultiSend address is available."""
+        multisend_address = kwargs.get("multisend_address", None)
+        enforce(
+            multisend_address is not None,
+            "'multisend_address' is a required parameter",
+        )
+        return multisend_address
```
Comment (0xArdi) [inline packages/balancer/skills/liquidity_provision_abci/models.py:84]: @DavidMinarsch this had to be added because multisend_address is necessary both in TerminationParams and LPParams. And since _ensure() utilizes kwargs.pop only one of them can make use of it.

File: packages/balancer/skills/autonomous_fund_abci/models.py (hunk 2 above)
Comment (DavidMinarsch) [reply packages/balancer/skills/autonomous_fund_abci/models.py:111]: Let's keep monitoring this and act once we have a convincing and transparent design

File: packages/balancer/skills/liquidity_provision_abci/models.py (hunk 3 above)
Comment (DavidMinarsch) [reply packages/balancer/skills/liquidity_provision_abci/models.py:84]: ok - another thing to monitor.

**Review by DavidMinarsch — APPROVED** [review]

**Review by DavidMinarsch — APPROVED** [review]
